kedro-airflow/kedro_airflow/plugin.py

```diff
@@ -91,8 +95,8 @@ def _get_pipeline_config(config_airflow: dict, params: dict, pipeline_name: str)
     "-t",
     "--target-dir",
     "target_path",
-    type=click.Path(writable=True, resolve_path=True, file_okay=False),
-    default="./airflow_dags/",
+    type=click.Path(writable=True, resolve_path=False, file_okay=False),
+    default="airflow_dags/",
```

Comment:
The target path shouldn't be resolved relative to the current directory at this point because it needs to be joined with the `metadata.project_path` later on.